### DISCUSSION TITLE
My PDS4 LDD Automation action failed, where can I find the generated LDDs?

### DISCUSSION THREAD

**Discussion by jordanpadams** [pds-data-dictionaries/PDS-Data-Dictionaries.github.io#28] [Q&A] [answered]
The github action failed on my repository, but I would still like to test with the LDDs that were generated to test them out locally. Can I find them somewhere?

<img width="1756" alt="Screenshot 2023-05-03 at 9 05 12 AM" src="https://user-images.githubusercontent.com/33492486/235974302-a78a67c2-c835-4a8b-b3b0-2e3283fd43c5.png">

**Comment by pds-ops** [comment, marked as answer]
Yes! They are uploaded to the GitHub Action artifacts for that specific execution. Scroll down on that page and you will see a ZIP of the LDDs can be downloaded there under `Artifacts`.

<img width="1758" alt="Screenshot 2023-05-03 at 9 06 50 AM" src="https://user-images.githubusercontent.com/93455202/235975101-d2f7dfc4-b7d1-4027-a1c8-cc54b7e9fab1.png">
